docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,608 +1,121 @@
-to use the newly created template to issue certificates used for Remote Desktop connections.
-
-### Create a certificate template
-
-In this section, we will create a certificate template that uses elliptic curve P-384 and uses SHA384 as the signature algorithm.
-
-
-1. Open the Microsoft Management Console (MMC)
-
-```text
-Start > Control Panel > Administrative Tools > Certificate Authority
-```
-
-2. Open your CA computer and right-click on `Certificate Templates`, then select `Manage`.
-3. Find the `Computer` template on the list, right-click on it, then select `Duplicate Template`.
-4. In the `Compatibility` tab change `Certification Authority` to `Windows Server 2012 R2` and click `OK`.
-5. In the same tab change `Certificate recipient` to `Windows Server 2012 R2` and click `OK`.
-6. Go to the `General` tab and change `Template display name` to `RemoteDesktopAccess`. Make sure `Template name` is also `RemoteDesktopAccess`.
-7. In the `Cryptography` tab change `Provider Category` to `Key Storage Provider`, then `Algorithm name` to `ECDH_P384`. Also, change `Request hash` to `SHA384`.
-8. Next, in the `Extensions` tab select `Application Polices` and click the `Edit` button.
-9. Remove all entries from the list.
-10. Go to the `Security` tab, select `Domain Controllers` and give the group `Read` and `Enroll` permissions.
-11. Finally, create a template by clicking `OK`.
-12. Go back to the Certificate Authority window and right-click on `Certificate Templates`. Then:
-
-```text
-New > Certificate Template to Issue
-```
-Select `RemoteDesktopAccess` and click `OK`.
-
-### Update GPO to use a new certificate template
-
-In the group policy editor for `Teleport Access Policy`, select:
-
-```
-Computer Configuration > Policies > Administrative Templates > Windows Components > Remote Desktop Services > Remote Desktop Session Host > Security
-```
-
-Right-click on `Server authentication certificate template`, `Edit`, then select `Enabled` and fill `Certificate Template Name` with `RemoteDesktopAccess`.
-
-<Figure align="left" bordered caption="RDP Certificate Template">
-  ![RDP Certificate Template](../../img/desktop-access/rdp-certificate-template.png)
-</Figure>
-
-
-### Configure server certificate auto-enrollment
-
-In the group policy editor for `Teleport Access Policy`, select:
-
-```
-Computer Configuration > Policies > Windows Settings > Public Key Policies
-```
-
-Double-click on `Certificate Services Client - Auto-Enrollment`, then select `Enabled` in the `Configuration Model`.
-
-
-### Ensure your GPO is updated
+</Details>
 
-If you have not done so already, ensure your GPO is updated by opening a PowerShell prompt and running:
+This guide will help you configure Teleport and your Active Directory Domain Controller to communicate securely with each other, populate available desktops, and provide secure access to them.
```

Comment:
```suggestion
This guide will help you configure Teleport and your Active Directory domain controller to communicate securely with each other, register available desktops with your Teleport cluster, and provide secure access to them.
```
- use Microsoft's capitalization of "domain controller"
- be a bit more explicit about what Teleport is doing with desktops